### DISCUSSION TITLE
Leemons local setup issues

### DISCUSSION THREAD

**Discussion by gulshanchouhan1993** [leemonade/leemons#82] [General]
Hello Leemons Team,

I am trying to setup the leemons project into my local machine to follow your provided documentation.
https://leemonade.github.io/leemons-docs/getting-started/installation

Did all the things according to the documentation and got 2 servers (i.e "core" and "front"). but when i tried to run it on my system browser.
The loader is shown and nothing happen. i got some error logs regarding the JWT tokens in my terminal. Also i got "<!doctype> is not a valid json" error in browser console as well.

Kindly help me out on the above errors and see the error attachments below for your reference.

![Screenshot from 2023-11-01 17-46-34](https://github.com/leemonade/leemons/assets/149657133/ae3e5531-d7b5-4149-a741-d6126da1b30d)

![Screenshot from 2023-11-02 13-03-28](https://github.com/leemonade/leemons/assets/149657133/bd20723d-c644-4655-8214-f5280caf899f)

Thanks

**Comment by MIGUELez11** [comment]
Hi!!

This error has been already reported in #81, We are looking at it :D
